vaas/vaas/router/models.py

```diff
@@ -6,6 +6,25 @@
 from vaas.cluster.models import LogicalCluster
 from vaas.manager.models import Director
 
+RESPONSE_STATUS_CHICES = (
+    ('301', 'Move Pernamentlly'),
+    ('302', 'Found'),
+    ('307', 'Temporary Redirect')
+)
+
+class Rewrite(models.Model):
+    condition = models.CharField(max_length=512)
+    destination = models.CharField(max_length=512)
+    action = models.CharField(max_length=64, choices=RESPONSE_STATUS_CHICES, default='301')
```

Comment:
Do we really need 64 characters for that field? Don't you think that int would be enough?

Reply:
I have been think thet this size also is connected whit tuple names but now i see the light. Three are enough.